R/carehomes.R

```diff
@@ -366,12 +366,16 @@ carehomes_parameters <- function(start_date, region,
   severity$psi_star <- severity$p_star / max(severity$p_star)
   severity$p_star_step <- max(severity$p_star)
 
-  strain_rel_gamma_A <- recycle(strain_rel_gamma_A, length(strain_transmission))
-  strain_rel_gamma_P <- recycle(strain_rel_gamma_P, length(strain_transmission))
-  strain_rel_gamma_C_1 <- recycle(strain_rel_gamma_C_1,
-                                  length(strain_transmission))
-  strain_rel_gamma_C_2 <- recycle(strain_rel_gamma_C_2,
-                                  length(strain_transmission))
+  strain_rel_gamma_A <- mcstate:::recycle(assert_relatives(strain_rel_gamma_A),
```

Comment:
for all of these can you please also add an assert non negative statement please? thx

Reply:
never mind seems to be there already - ignore me